templates/zerver/help/start-a-call.md

```diff
@@ -77,3 +94,4 @@ setting the provider to "None".
 
 [big-blue-button-configuration]: https://zulip.readthedocs.io/en/latest/production/video-calls.html#big-blue-button
 [zoom-configuration]: https://zulip.readthedocs.io/en/latest/production/video-calls.html#zoom
+[webex-configuration]: https://zulip.readthedocs.io/en/latest/production/video-calls.html#webex
```

Comment:
I think you forgot to add the documentation in `docs/production/video-calls.md` that this links to.

Reply:
Thanks, I have added it now.